PDF/demerge_pdfs.py

```diff
@@ -0,0 +1,43 @@
+"""
+Python program to split large pdf(typically textbook) into small set of pdfs, maybe chapterwise
+to enhance the experience of reading and feasibility to study only specific parts from the large original textbook
+"""
+
+
+import PyPDF2
+path = input()
```

Comment:
**It would be better if a function is used to encapsulate all this program statements.**
**Also, use __name__ == '__main__' statement to run the program.
Would be better for importing.